### ISSUE TITLE
Switch to hugo pipes?

### ISSUE THREAD

**Issue by nidomiro** [Lednerb/bilberry-hugo-theme#238]
Have you thought of switching to hugo pipes (https://gohugo.io/hugo-pipes/) for scss and js packing?
I think that approach looks very promising.

My Plan was to switch to hugo pipes, but only if you support the move (and with support I mean "You are OK with that" ;) )

**Comment by Lednerb** [comment]
Hey @nidomiro 

I've used hugo pipes in a previous version of this theme before https://github.com/Lednerb/bilberry-hugo-theme/commit/0197b17d328a291a30e9aa02f37472f11debd2e2.

It was a cool option and I've also integrated some SCSS-Processing with the hugo config, for example for adjusting the site width etc.
However, dependency management was a problem because there is none integrated and the code was not very clean in terms of having one place to config all related settings.

Also the users of this theme had to install the `extended` version of hugo instead of having the `simple` one.

---

I've decided to switch back to the `npm` package management and having webpack / laravel-mix for an easy way to build the assets, but I also don't like to install / download hundreds of packages for processing a handful of `SCSS` and `JS` files.

-----

TL;DR:
I don't plan to use the hugo pipes feature in the future again.
